pytorch_lightning/plugins/training_type/single_hpu.py

```diff
@@ -0,0 +1,24 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from pytorch_lightning.strategies import HPUStrategy
+from pytorch_lightning.utilities import rank_zero_deprecation
+
+
+class HPUPlugin(HPUStrategy):
```

Comment:
Let's not introduce `HPUPlugin`, as it's a new strategy and importing `HPUStrategy` would be the right way for the users.